client/components/olark-chat/index.jsx

```diff
@@ -1,39 +1,33 @@
+import PropTypes from 'prop-types';
 import { Component } from 'react';
+import OlarkChatMain from './main';
+import OlarkErrorBoundary from './olark-error-boundary';
 
 class OlarkChat extends Component {
-	componentDidMount() {
+	render() {
```

Comment:
Nit: This could now be nicely and easily converted to a functional component.

Reply:
I agree! I've made this change in [13bb0f1](https://github.com/Automattic/wp-calypso/pull/62483/commits/13bb0f1ae30ae5f666ecfec7cc858f9b63f5a2f6).